SampleUserControlLibrary/SubscriptionKeyPage.xaml.cs

```diff
@@ -49,7 +49,7 @@ public partial class SubscriptionKeyPage : Page, INotifyPropertyChanged
         private readonly string _isolatedStorageSubscriptionEndpointFileName = "SubscriptionEndpoint.txt";
 
         private readonly string _defaultSubscriptionKeyPromptMessage = "Paste your subscription key here firstly";
-        private readonly string _defaultSubscriptionEndpointPromptMessage = "Paste your endpoint here to start";
+        private string _defaultSubscriptionEndpointPromptMessage = "Paste your endpoint here to start";
```

Comment:
We do not need to change the **default** endpoint setting. Just set the endpoint should be enough. Just make the default default.